editor/store/effects.js

```diff
@@ -489,7 +492,7 @@ export default {
 		}, {} );
 		store.dispatch( setMetaBoxSavedData( dataPerLocation ) );
 	},
-	REQUEST_META_BOX_UPDATES( action, store ) {
+	REQUEST_META_BOX_UPDATES( { post }, store ) {
```

Comment:
Does `post` need to come through the action, or could we assume that the value in `store.getState()` would be up to date by now?

Reply:
Yes, it's probably the same, I'll remove the post from the action

Reply:
Is it? I mean, it might be the same. It all depends how all those `dispatch` actions are processed. This is quite tricky because you dispatch action in the middle of another dispatch. In the moment when you trigger `requestMetaBoxUpdates` action, the payload of `requestPostUpdateSuccess` is still processed, so there might be a different value in the store at that time. I think actions are dispatched asynchronously which means it should all end up as expected.